land-ownership/README.md

```diff
@@ -19,9 +19,9 @@ interested specifically in land-owning companies and their land.
 Most land-owning companies are small and simple, but others exhibit complex legal structures as a result of
 company mergers and acquisitions, as well as by design (for legal/financial reasons).
 
-For example, Sainsburys (the supermarket) _indirectly_ owns several thousand parcels of land in the UK via
+For example, Sainsburys (the supermarket) _indirectly_ owns several thousand parcels of land in the UK through ownership of
 sub-companies and sub-sub-companies. However its top-level legal entity, "J Sainsbury PLC", only owns about 40
-parcels _directly_.
+parcels _directly_. Note - indirect ownership is one-way; these 40 parcels are _not_ indirectly owned by any sub-companies.
```

Comment:
I wonder if that might confuse people who already understood it. How about putting it in parens, and a "to be clear" to make it clear that this isn't really new info.

(To be clear, a given company is said to _indirectly_ own land if one of its sub-companies, or sub-sub...-companies owns the land, but this relationship only goes one-way)

Reply:
I'm agnostic, happy to leave you both to discuss and figure out a text amend if you feel it's needed